lib/data_source/client/github.rb

```diff
@@ -7,17 +7,30 @@
 module DataSource
   module Client
     class Github
-      CACHE_TTL_SECS = 3600 * 12
 
       def initialize(config)
         @host = config[:host]
         @access_token = config[:access_token]
         @cache_dir = config[:cache_dir]
+        @me_account = config[:me_account]
+        @pr_all_involve_me = config[:pr_all_involve_me]
+
+        @cache_name_hash = {
+            user_repos: 'user_repos',
+            user_orgs: 'user_orgs',
+            user_pulls: 'user_pulls'
+        }
+
+        @cache_ttl_hash = {
+            user_repos: config[:cache_ttl_sec_repo],
+            user_orgs: config[:cache_ttl_sec_org],
+            user_pulls: config[:cache_ttl_sec_pr]
+        }
```

Comment:
Now that we have more options in the configuration, let's only assign the config hash as an instance variable and access all the individual key from it.

```ruby
def initialize(config)
  @config = config
end
```

Another refactor for later could be moving all the configuration to its own class in `DataSource::ClientConfiguration`